frontend/simple.js

```diff
@@ -0,0 +1,110 @@
+/**
+ * Created by timadamson on 8/22/17.
+ */
+
+/**
+ * An App self adds 3 camera streams, and the controlers to move the robot
+ */
+
+App = function () {
```

Comment:
This file defines an App function. What we want is to create an instance of an App. I can add a simple.js file that does that if you want.

Reply:
pls do